cmd/trader/trader.go

```diff
@@ -34,146 +41,232 @@ func main() {
 	// Parse the option parameters
 	numberOfOrders := flag.Int("order", 10, "number of orders")
 	timeInterval := flag.Int("time", 15, "time interval in second")
+	cancelRequest := flag.Bool("cancel", false, "cancel request")
```

Comment:
call flag.Parse() after all flags have been defined

Reply:
Thanks, @tok-kkk . Will look into this now.